README.md

```diff
@@ -125,7 +151,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+Schema is a collection of database objects that is linked with a specific database. Schemas are usually tables with data that are interrelated with each other which explains the one-to-many relationship: Every row of data in the table is somehow related to the other rows of data in the table.
 ```
```

Comment:
I'd say a schema is a blueprint for a table, i.e. which columns it has. It can define a collection of object-like structures for a database, but they are not truly objects until you involve an ORM like Active Record, for instance